DailyGoals/DailyGoals/Utilities/DailyGoalsGenerator.swift

```diff
@@ -7,21 +7,44 @@
 
 import SwiftUI
 
-public struct DailyGoalsGenerator {
+struct DailyGoalsGenerator {
     
-    public static func text(goals: [Goal]) {
+    static func text(goals: [Goal]) {
         generate(goals)
     }
     
     private static func generate(_ goals: [Goal]) {
-        var text = "@squid *Daily Goals\(DateFormatter.formattedDate())* \(SquidEmojis.squids.randomElement()!)\n"
+        var text = "@squid *Daily Goals \(DateFormatter.formattedDate())* \(SquidEmojis.squids.randomElement()!)\n"
         
         for goal in goals {
-            text += ":circle: [HSC-\(goal.ticketNumber)](https://hudl-jira.atlassian.net/browse/HSC-\(goal.ticketNumber)) - \(goal.description) - \(Team.members[goal.assignedPerson] ?? "") - \(goal.goalAction)\n"
+			text += goal.text
         }
         
         let pasteboard = NSPasteboard.general
         pasteboard.clearContents()
         pasteboard.setString(text, forType: .string)
     }
 }
+
+private extension Goal {
+
+	var text: String {
+		var text = ":circle: [HSC-\(ticketNumber)](https://hudl-jira.atlassian.net/browse/HSC-\(ticketNumber))"
+
+		if description.isEmpty == false {
+			text += " - \(description)"
```

Comment:
I wanted to make sure it's not empty before we copy, but I guess this can stay for now